openerp/openupgrade/tests_deferred/tests_deferred.py

```diff
@@ -14,3 +14,9 @@ def test_binary_attachments(self):
                 ('res_field', '=', 'image'),
                 ('res_name', '=', 'Axelor...'),
             ]).datas)
+
+    def test_hr_attendance_view_removed(self):
+        """ This view, which had the noupdate flag, is now successfully removed
+        """
+        with self.assertRaisesRegexp(ValueError, 'External ID not found'):
```

Comment:
For me, it's simpler to check existence with this code, but it's only FYI:
```
view = self.env.ref('hr_attendance.hr_attendace_group', False)
self.assertFalse(view)
```

Reply:
It's equivalent, indeed. I thought using `with assertRaises` would make me look smart ;-)